app/bundles/FormBundle/Views/Field/field_helper.php

```diff
@@ -161,6 +161,16 @@
         }
     }
 
+    if ($field['leadField'] && in_array($contactFields[$field['leadField']]['type'], ['datetime', 'date'])) {
+        $tempLeadFieldType = $contactFields[$field['leadField']]['type'];
+        $mauticParameter   = $this->container->getParameter('mautic.parameters');
+        foreach ($parseList as $key => $aTemp) {
+            $date = date_create_from_format('Y-m-d', $aTemp['label']);
```

Comment:
Hi thanks for review.
Code has been updated with requested changes. Please check now.
I think the value needs to be in format Y-m-d because the form field bind with the lead field so it will be updated easily.